x-pack/test/apm_api_integration/tests/throughput/service_maps.spec.ts

```diff
@@ -113,7 +121,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             [
               ...Object.values(throughputTransactionValues),
               ...Object.values(throughputMetricValues),
-            ].forEach((value) => expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE)));
+            ].forEach((value) =>
+              expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE + GO_PROD_RATE))
+            );
+          });
+        });
+
+        describe('when calling service maps API', () => {
```

Comment:
can you make this more descriptive, to signal the fact that it is calling the throughput API? Currently the test name looks like `Service maps APIs > when calling service maps API` (I think)